README.md

```diff
@@ -56,20 +70,7 @@ Consider increasing maximum heap size for the IDE (the default value is 2048 MB)
 
 For running `./gradlew` from command line, make sure that `java` and `javac` are in `PATH` and point to Java 11.
 
-For running tests of `backendImpl` (which are included under `./gradlew test`, which is in turn itself included under `./gradlew build`),
-install [`git-machete` CLI](https://github.com/VirtusLab/git-machete#install) (preferably via `pip install git-machete==<version>`)
-in the version present in [backendImpl/src/test/resources/reference-cli-version.properties](backendImpl/src/test/resources/reference-cli-version.properties).
-
-#### Optional
-
-[Grammar-Kit IntelliJ plugin](https://plugins.jetbrains.com/plugin/6606-grammar-kit) can be used instead of Gradle plugin
-to manually generate grammar and lexer code from `.bnf` and `.flex` files.
-
-When running IntelliJ instance with a plugin loaded then [PsiViewer IntelliJ plugin](https://plugins.jetbrains.com/plugin/227-psiviewer)
-can be helpful to see parsing result on machete file.
-
-
-### Build
+### Building
```

Comment:
Hmm this part is (I guess intentionally) repeated in DEVELOPMENT.md... TBH I don't think we need details like that in README, it just unnecessarily bloats our "front page" and can have somewhat intimidating effect (at least I don't think it's gonna be helpful); just a link to a full-fledged development guide should be enough in README

Reply:
Yeah, and definitely this should not be before the `Installing` section :scream_cat:

Reply:
About `Building` section, as it's an opensorce project I think that basic information about how to build it should be visible directly in `README.md` because it's natural that someone want to build that plugin on they own. Because of that I also think that moving all build instructions into `DEVELOPMENT.md` only is not a good move - IMO it's less intuitive (but maybe it's only my opinion).

Reply:
Ok, I agree with this reasoning. If it comes to `In case of spurious cache-related...` paragraph, pls either remove it completely or update to match the one in DEVELOPMENT.md